src/server/mom_info.c

```diff
@@ -503,6 +505,209 @@ delete_svrmom_entry(mominfo_t *pmom)
 	delete_mom_entry(pmom);
 }
 
+/**
+ * @brief
+ *	Find the pool that matches what is set on the node
+ *
+ * @param[in]	pmom - pointer to the mom
+ * @param[out]	ppool - pointer to the matching pool structure
+ *
+ * @return	vnpool_mom_t *
+ * @retval	pointer to the matching pool structure
+ * @retval	NULL - if there is no match
+ */
+vnpool_mom_t *
+find_vnode_pool(mominfo_t *pmom)
+{
+	mom_svrinfo_t *psvrmom = (mom_svrinfo_t *)(pmom->mi_data);
+	vnpool_mom_t *ppool = vnode_pool_mom_list;
+
+	if (psvrmom->msr_vnode_pool != 0) {
+		while (ppool != NULL) {
+			if (ppool->vnpm_vnode_pool == psvrmom->msr_vnode_pool) {
+				return(ppool);
+			}
+			ppool = ppool->vnpm_next;
+		}
+	}
+	return NULL;
+}
+
+/**
+ * @brief
+ *	Reset the "inventory Mom" for a vnode_pool if the specified Mom is the
+ *	current inventory Mom.  Done when she is down or deleted from the pool.
+ *
+ * @param[in] pmom - Pointer to the Mom (mominfo_t) structure of the Mom
+ *	being removed/marked down.
+ */
+void
+reset_pool_inventory_mom(mominfo_t *pmom)
+{
+	mom_svrinfo_t *psvrmom = (mom_svrinfo_t *)(pmom->mi_data);
+	vnpool_mom_t  *ppool;
+	mominfo_t     *pxmom;
```

Comment:
remove extra spaces

Reply:
extra spaces help it stand aligned with the previous line.